script.js

```diff
@@ -116,3 +116,47 @@ const btn = document.querySelectorAll('.btn');
 btn.forEach((element) => {
   element.addEventListener('click', removeBk);
 });
+
+// navbar section
+
+const links = document.querySelectorAll('.links');
+
+function showBlock(e) {
+  const sectionList = document.getElementById('list');
+  const sectionAdd = document.getElementById('add_book');
+  const sectionContact = document.getElementById('contact');
+  switch (e) {
+    case 'list-link':
+      sectionList.style.display = 'block';
+      sectionAdd.style.display = 'none';
+      sectionContact.style.display = 'none';
+      break;
+
+    case 'add-link':
+      sectionAdd.style.display = 'block';
+      sectionList.style.display = 'none';
+      sectionContact.style.display = 'none';
+      break;
+
+    case 'contact-link':
+      sectionContact.style.display = 'flex';
+      sectionAdd.style.display = 'none';
+      sectionList.style.display = 'none';
+      break;
+
+    default:
+      sectionAdd.style.display = 'block';
+      sectionList.style.display = 'none';
+      sectionContact.style.display = 'none';
+      break;
+  }
+}
```

Comment:
- [ ] When I click on the `Add new` or `contact` option from the navigation I get a blank page on the screen. Please fix this issue and make sure that the functionalities of the app are working as expected.
![image](https://user-images.githubusercontent.com/83803414/197857918-55c5fc3e-d007-4e1f-9702-dc1e20605a3d.png)